src/olympia/constants/activity.py

```diff
@@ -545,6 +545,14 @@ class SOURCE_CODE_UPLOADED(_LOG):
     review_queue = True
 
 
+class CONFIRM_AUTO_APPROVED(_LOG):
+    id = 144
+    format = _(u'Auto-Approval confirmed for {addon} {version}.')
+    short = _(u'Auto-Approval confirmed')
+    keep = True
+    editor_review_action = True
+
```

Comment:
You might need `review_queue` in here too to make it show up in the editors review history (the various properties are, unsurprisingly, not documented).